pkgs/development/python-modules/korean-lunar-calendar/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "korean-lunar-calendar";
+  version = "0.2.1";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "korean_lunar_calendar";
+    sha256 = "0p97r21298ipgvsqh978aq2n6cvybzp8bskcvj15mm1f76qm9khj";
+  };
+
+  # no real tests
```

```suggestion
  # no real tests
  doCheck = false;
```
Maybe disable tests entirely then? Unsure what ``setup.py test`` would do...

---

It just finds zero tests. I'd like to think though that if left "enabled", when the package does gain tests it would spring into action and either work or nudge someone to get them working.

---

Also, if we set `doCheck = false` on all packages in nixpkgs that had no tests, we'd have a few million occurrences of that.